README.md

```diff
@@ -1,6 +1,6 @@
 # Llama 2 Fine-tuning / Inference Recipes, Examples and Demo Apps
 
-**[Update Nov. 3, 2023] We recently released a series of Llama 2 demo apps [here](./demo_apps). These apps show how to run Llama 2 locally, in the cloud, and on-prem, and how to ask Llama 2 questions in general and about custom data (PDF, DB, or live).**
+**[Update Nov. 16, 2023] We recently released a series of Llama 2 demo apps [here](./demo_apps). These apps show how to run Llama 2 locally, in the cloud, on-prem, how to ask Llama 2 questions in general or about custom data (PDF, DB, or live) and end-to-end Chatbot implementation with RAG (Retrieval Augmented Generation).**
```

Comment:
how to run Llama 2 locally, in the cloud, on-prem, how to ask Llama 2 questions in general or about custom data (PDF, DB, or live) and end-to-end Chatbot implementation
->
how to run Llama (locally, in the cloud, or on-prem), how to ask Llama questions in general or about custom data (PDF, DB, or live), how to integrate Llama with WhatsApp, and how to implement an end-to-end chatbot

Reply:
Done, should be all set now!